index.js

```diff
@@ -92,18 +92,23 @@ Merry.prototype.start = function () {
   return this._router
 }
 
-Merry.prototype.listen = function (port) {
+Merry.prototype.listen = function (port, cb) {
   assert.ok(this._router, 'merry.listen: router was not found. Did you run app.router() ?')
   var self = this
   var server = http.createServer(this._router)
 
+  cb = cb || noop
```

Comment:
hmmm that's interesting, you just have an blank cb func? I get it, but a lil confusing on first couple of glances i suppose 

a+ on adding a cb though